dm/ctl/ctl.go

```diff
@@ -131,11 +132,12 @@ func PrintHelp(args []string) {
 }
 
 // Start starts running a command
-func Start(args []string) {
+func Start(args []string) (err error) {
 	commandMasterFlags.Reset()
 	rootCmd = NewRootCmd()
 	rootCmd.SetArgs(args)
-	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(rootCmd.UsageString())
+	if err = rootCmd.Execute(); err != nil {
+		//fmt.Println(rootCmd.UsageString())
```

Comment:
remove it?

Reply:
https://github.com/pingcap/dm/pull/845/commits/45742af81173d51efc763299ddb08d56a3a046aa

After changing to `RunE`, `rootCmd.Execute()` will return error now. For some error we manually print usage inside command function, so don't print again here (will break some tests)

Reply:
forget to set returned err, fix here https://github.com/pingcap/dm/pull/845/commits/61456ff3b52c870dcf2a80ab365e453dc2c2e14f